package.json

```diff
@@ -55,23 +55,23 @@
     "main": "./main.js",
     "contributes": {
         "javaExtensions": [
-            "./server/junit-jupiter-api_5.9.1.jar",
-            "./server/junit-jupiter-engine_5.9.1.jar",
-            "./server/junit-jupiter-migrationsupport_5.9.1.jar",
-            "./server/junit-jupiter-params_5.9.1.jar",
-            "./server/junit-platform-commons_1.9.1.jar",
-            "./server/junit-platform-engine_1.9.1.jar",
-            "./server/junit-platform-launcher_1.9.1.jar",
-            "./server/junit-platform-runner_1.9.1.jar",
-            "./server/junit-platform-suite-api_1.9.1.jar",
-            "./server/junit-platform-suite-commons_1.9.1.jar",
-            "./server/junit-platform-suite-engine_1.9.1.jar",
-            "./server/junit-vintage-engine_5.9.1.jar",
+            "./server/junit-jupiter-api_5.9.2.jar",
+            "./server/junit-jupiter-engine_5.9.2.jar",
+            "./server/junit-jupiter-migrationsupport_5.9.2.jar",
+            "./server/junit-jupiter-params_5.9.2.jar",
+            "./server/junit-platform-commons_1.9.2.jar",
+            "./server/junit-platform-engine_1.9.2.jar",
+            "./server/junit-platform-launcher_1.9.2.jar",
+            "./server/junit-platform-runner_1.9.2.jar",
+            "./server/junit-platform-suite-api_1.9.2.jar",
+            "./server/junit-platform-suite-commons_1.9.2.jar",
+            "./server/junit-platform-suite-engine_1.9.2.jar",
+            "./server/junit-vintage-engine_5.9.2.jar",
             "./server/org.apiguardian.api_1.1.2.jar",
             "./server/org.eclipse.jdt.junit4.runtime_1.3.0.v20220609-1843.jar",
             "./server/org.eclipse.jdt.junit5.runtime_1.1.100.v20220907-0450.jar",
-            "./server/org.opentest4j_1.2.0.v20211018-1956.jar",
-            "./server/com.microsoft.java.test.plugin-0.37.1.jar"
+            "./server/org.opentest4j_1.2.0.v20190826-0900.jar",
```

Comment:
looks like the timestamp of this jar is downgrading?

Reply:
Good finding! PR is updated.